server/service.go

```diff
@@ -366,6 +369,11 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 			// Remember which relays delivered which bids (multiple relays might deliver the top bid)
 			relays[BlockHashHex(blockHash)] = append(relays[BlockHashHex(blockHash)], relay)
 
+			// Skip if value (fee) is lower than the minimum bid
```

Comment:
move this to line 365, above saving the relays and locking

Reply:
done